webpack/InsightsCloudSync/Components/InsightsSettings/InsightsSettings.js

```diff
@@ -12,9 +12,17 @@ class InsightsSettings extends Component {
 
   render() {
     const { insightsSyncEnabled, setInsightsSyncEnabled } = this.props;
+    const insightsStyles = {
+      border: '1px solid #ededed',
```

Comment:
I am a bit reluctant about moving colors to inline style. I suppose it would be harder to themify it.
Maybe we should have some class or global  constant for it, so it would be easier to themify it in the same place?

Reply:
good point.. I tried to do fetch the plugin css file on the global-js file (in the next foreman version it will be available from core)
though there is a delay in the view so you see the page breaking and changing (this happens only if you refresh the insights page) if you come from another page the styles will be already predefined.